### DISCUSSION TITLE
None of Vertical Tabs packages I tried, works realiable in Pulsar

### DISCUSSION THREAD

**Discussion by gotenks** [pulsar-edit/.github#261] [Community Package Help]
Hi,
I tried four of them, but none of them activates automatically after Pulsar restarts. I need to Disable/Enable them from Settings, to have vertical tabs again. And often even during normal work, Pulsar turns off vertical tabs, switching to horizontal ones. Any suggestion ?

**Comment by savetheclocktower** [comment]
Which ones have you tried?

**Comment by gotenks** [comment]
vertical-tabs 1.0.7
vertical-tabs-plus 2.4.0
vertical-tabs -right 1.1.0
resizable-vertical-tabs 2.2.0
